### PR TITLE
adding freebasics data model and migrations

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony model and migration ready for review

File: freebasics/models.py [hunk 1]
```diff
@@ -22,3 +22,21 @@ def app_id(self):
         The app id to use for marathon
         """
         return 'freebasics-%s' % self.slug
+
+
+class FreeBasicsTemplateData(models.Model):
+    siteName = models.CharField(max_length=100, blank=True, null=True)
```
Comment (miltontony) [inline freebasics/models.py]: naming convention - please use `variable_name`

Comment (miltontony) [inline freebasics/models.py]: Also, I think we should make both `site_name` and `site_name_url` unique

File: freebasics/models.py [hunk 2]
```diff
@@ -22,3 +22,21 @@ def app_id(self):
...
+    siteNameUrl = models.CharField(
```
Comment (miltontony) [inline freebasics/models.py]: This should be a `URLField` with `max_length=255`

**Comment by codiebeulaine** [comment]
@miltontony fixed naming convention

**Comment by miltontony** [comment]
:+1: create migrations and we're good to go

**Pull request merged**